index.bs

```diff
@@ -4365,6 +4366,48 @@ navigated</dfn> steps given |navigable| and |navigation status|:
 
 </div>
 
+#### The browsingContext.historyUpdated Event #### {#event-browsingContext-historyUpdated}
+
```

Comment:
I know we don't have this for the other events, but it could be useful to have some informative text indicating that this is only fired for `pushState` and friends, not for history updates caused by navigation.

Reply:
The discussion on the HTML spec resulted in this event being sent whenever there is a user-observable history update according to the HTML spec so I think we probably do not need a note but I can add it as a follow-up.